Tilde.Translation/Translator.cs

```diff
@@ -39,6 +39,16 @@ public Translator(string apiKey, TranslatorOptions? options = null)
         {
             options ??= new TranslatorOptions();
 
+            if (string.IsNullOrWhiteSpace(options.AppInfo?.AppVersion))
```

Comment:
If empty values throws exceptions, why properties are marked as nullable in model class?